docs/development/extensions-core/hdfs.md

```diff
@@ -53,3 +51,34 @@ The HDFS extension can also be used for GCS as deep storage.
 All services that need to access GCS need to have the [GCS connector jar](https://cloud.google.com/hadoop/google-cloud-storage-connector#manualinstallation) in their class path. One option is to place this jar in <druid>/lib/ and <druid>/extensions/druid-hdfs-storage/
 
 Tested with Druid 0.9.0, Hadoop 2.7.2 and gcs-connector jar 1.4.4-hadoop2.
+
+<a name="firehose"></a>
+
+## Native batch ingestion
+
+This firehose ingests events from a predefined list of S3 objects.
```

Comment:
Heh, your copy and paste 🍝 is showing 😅, you mean HDFS instead of S3?

Reply:
Fixed in https://github.com/apache/incubator-druid/pull/8767